validphys2/src/validphys/pdfbases.py

```diff
@@ -34,6 +34,23 @@
         (21 , r"g"),
     ))
 
+PIDS_DICT = {
```

Comment:
@andreab1997 @felixhekhorn consider that we have something extremely similar in EKO (`eko.basis_rotation`), so I don't know if it's worth to try to reuse those objects (or merge the two, in some sense).

Reply:
Just remember to take this into account